asmcomp/arm64/emit.mlp

```diff
@@ -683,6 +688,20 @@ let name_for_float_comparison = function
 
 (* Output the assembly code for an instruction *)
 
+let emit_stlr src base addr =
+  assert macosx;
+  let dest_reg =
+    match addr with
+    | Iindexed 0 -> base
+    | Iindexed ofs ->
+        `	add	{emit_reg reg_tmp1}, {emit_reg base}, #{emit_int ofs}\n`;
+        reg_tmp1
+    | Ibased _ -> assert false (* Ibased is not emitted under macOS *)
+  in
+  (* Release store for assignments on macOS. See
+     https://github.com/ocaml/ocaml/issues/13262. *)
```

Comment:
This comment shoud go to line 870, where `emit_stlr` is called to handle `Istore` in the `true, true` case.  The comment is not relevant  to the `emit_stlr` function itself, but to its use for implementing `Istore`.